### PR TITLE
Rewrite intro section of custom data

### PR DESCRIPTION
**Pull request by kmoscoe**
Before the full-on variable per row coverage, I found some egregiously incorrect statements, so doing an incremental update here. (Also cleaning up some TOC levels.)

### CODE REVIEW

File: custom_dc/custom_data.md [hunk 1]
```diff
@@ -17,14 +17,49 @@ This page shows you how to format and load your own custom data into your local
...
+| San Francisco | 2023 | private | middle | 100 |
+| San Francisco | 2023 | private | secondary | 50 |
+| San Jose | 2023 | public | elementary | 400 |
+| San Jose | 2023 | public | middle | 400 |
+| San Jose | 2023 | public | secondary | 300 |
+| San Jose | 2023 | private | elementary | 200 |
+| San Jose | 2023 | private | middle | 200 |
+| San Jose | 2023 | private | secondary | 100 |
+
+Although the properties of school type and school level may already be defined in the Data Commons knowledge graph (or you may need to define them), they _cannot_ be present as columns in the CSV files that you store in Data Commons. Instead, you must create separate "count" variables to represent each case. In our example, you would actually need 6 different variables:
```
Comment (keyurva) [inline custom_dc/custom_data.md:53]: Can we caveat this by saying that the schema allows for it (same variable with different observation properties) but with simple CSV formats for custom dcs they should define separate variables.

File: custom_dc/custom_data.md [hunk 2]
```diff
@@ -17,14 +17,49 @@ This page shows you how to format and load your own custom data into your local
...
 
+We strongly recommend that, before proceeding, you familiarize yourself with the basics of the Data Commons data model by reading through [Key concepts](/data_model.html), in particular, _entities_, _statistical variables_, and _observations_.
+
+The following sections walk you through the process of setting up your data.
+
+## Before you start: Identify your statistical variables
+
+Your data undoubtedly contains metrics and observed values. In Data Commons, the metrics themselves are known as statistical variables, and the time series data, or values over time, are known as observations. While observations are always numeric, statistical variables must be defined as _nodes_ in the Data Commons knowledge graph.  
+
+Statistical variables must follow a certain model; in particular, they must represent any breakdown properties and even encode those properties in their name. To explain what this means, consider the following example. Let's say your dataset contains the number of schools in U.S. cities, broken down by level (elementary, middle, secondary) and type (private, public), reported for each year (numbers are not real, but are just made up for the sake of example)::
```
Comment (keyurva) [inline custom_dc/custom_data.md:36]: nit: Remove the extra colon at the end of the sentence.

File: custom_dc/custom_data.md (hunk 1 above)
Comment (kmoscoe) [reply custom_dc/custom_data.md:53]: Let's leave it for now; I will include with the rewrite for variable-per-row.

File: custom_dc/custom_data.md (hunk 2 above)
Comment (kmoscoe) [reply custom_dc/custom_data.md:36]:
```suggestion
Statistical variables must follow a certain model; in particular, they must represent any breakdown properties and even encode those properties in their name. To explain what this means, consider the following example. Let's say your dataset contains the number of schools in U.S. cities, broken down by level (elementary, middle, secondary) and type (private, public), reported for each year (numbers are not real, but are just made up for the sake of example):
```

File: custom_dc/custom_data.md [hunk 3]
```diff
@@ -17,14 +17,49 @@ This page shows you how to format and load your own custom data into your local
...
+| San Francisco | 2023 | private | secondary | 50 |
+| San Jose | 2023 | public | elementary | 400 |
+| San Jose | 2023 | public | middle | 400 |
+| San Jose | 2023 | public | secondary | 300 |
+| San Jose | 2023 | private | elementary | 200 |
+| San Jose | 2023 | private | middle | 200 |
+| San Jose | 2023 | private | secondary | 100 |
+
+Although the properties of school type and school level may already be defined in the Data Commons knowledge graph (or you may need to define them), they _cannot_ be present as columns in the CSV files that you store in Data Commons. Instead, you must create separate "count" variables to represent each case. In our example, you would actually need 6 different variables:
+- `CountPublicElementary`
```
Comment (dwnoble) [inline custom_dc/custom_data.md:54]: Rename these variables like `Count_Public_Elementary` for consistency with other data commons variables

Comment (kmoscoe) [reply custom_dc/custom_data.md:54]: Done

**Review by keyurva — APPROVED** [review]

**Pull request merged**